src/classes/LightningMetadataController_Tests.cls

```diff
@@ -0,0 +1,34 @@
+@isTest
+private class LightningMetadataController_Tests {
+
+    @isTest
+    static void getSObjectMetadataTest() {
+        String sobjectName = 'Account';
+        DescribeSObjectResult sobjectDescribe = Schema.Account.SObjectType.getDescribe();
+
+        Test.startTest();
+        LightningMetadataController.SObjectMetadata sobjectMetadata = LightningMetadataController.getSObjectMetadata(sobjectName);
+        Test.stopTest();
+
+        System.assertEquals(sobjectName, sobjectMetadata.name);
+        System.assertEquals(sobjectDescribe.getLabel(), sobjectMetadata.label);
+        System.assertEquals(sobjectDescribe.getLabelPlural(), sobjectMetadata.labelPlural);
+    }
+
+    @isTest
+    static void getFieldMetadataTest() {
```

Comment:
Method `getFieldMetadataTest` doesn't follow naming convention. Please rename it using naming convention: Lower camel case with underscores (i.e. myName_with_moreParts).